nf_core/pipelines/rocrate.py

```diff
@@ -287,7 +287,7 @@ def add_main_authors(self, wf_file: rocrate.model.entity.Entity) -> None:
         try:
             git_contributors: Set[str] = set()
             if self.pipeline_obj.repo is None:
-                log.info("No git repository found. No git contributors will be added as authors.")
+                log.debug("No git repository found. No git contributors will be added as authors.")
```

Comment:
that can stay on info, imo

Reply:
When we lint a pipeline we create a template in a temporary directory, to compare the files that don't have to change. On this temp directory we don't have a github repo, so this message is always printed on linting. I couldn't think of a better way to hide the message there.